src/main/resources/application.properties

```diff
@@ -83,6 +85,7 @@ quarkus.hibernate-orm.database.generation=drop-and-create
 %dev.quarkus.datasource.min-size=3
 %dev.quarkus.datasource.max-size=13
 %dev.quarkus.hibernate-orm.database.generation=drop-and-create
+%dev.quarkus.hibernate-orm.sql-load-script=import.sql
```

Comment:
I think that this is not really needed, as its the default configuration anyway.
Also, since import.sql is loaded by hibernate lets remove it from the datasource url, as its causing issues.